README.md

```diff
@@ -1 +1,38 @@
-# go-clang-phoenix-bootstrap
+# go-clang-phoenix-bootstrap [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-bootstrap-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-bootstrap?branch=master)
+
+Native Go bindings for the C API of Clang.
+
+#Usage
+
+There is no direct way to use this repo other than for bootstrapping new go-clang-bindings. 
+Look over to [gen](https://github.com/zimmski/go-clang-phoenix-gen) to discover how to generate a new go-clang-binding.
+
+## Where are the bindings?
+
+The Go bindings are placed in their own repositories to provide the correct bindings for the corresponding Clang version:
+
+- [v3.4](https://github.com/zimmski/go-clang-phoenix-v3.4)
+- [v3.6](https://github.com/zimmski/go-clang-phoenix-v3.6)
+- [v3.7](https://github.com/zimmski/go-clang-phoenix-v3.7)
+
+## Where is the generation?
+
+The go-clang binding generation is in its own repository and can be found at [gen](https://github.com/zimmski/go-clang-phoenix-gen).
+
+## Installation
+
+```bash
+CGO_CFLAGS="-I`llvm-config --includedir`" \
+  CGO_LDFLAGS="-L`llvm-config --libdir`" \
+  go get github.com/zimmski/go-clang-phoenix-bootstrap
+```
+
+## Example
+
+An example on how to use the AST visitor of Clang can be found in [/cmd/go-clang-dump/main.go](/cmd/go-clang-dump/main.go)
+
+## Issue Tracking
```

Comment:
Please use `I found a bug/missing a feature in go-clang` here which is more explicit.